src/common/transformations/include/transformations/op_conversions/reduce_modification_for_false_keepdims.hpp

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <memory>
+#include <openvino/core/rt_info.hpp>
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/pass/graph_rewrite.hpp>
+#include <transformations_visibility.hpp>
+#include <vector>
+
+namespace ov {
+namespace pass {
+
+// Modify reduce if keep_dims is false and it reduces batch and spatial axes
+class TRANSFORMATIONS_API ModifyReduceForFalseKeepDims;
```

Comment:
please add some tests to https://github.com/openvinotoolkit/openvino/tree/master/src/common/transformations/tests

Reply:
Added

Reply:
Did you upload the test? I can't find..